typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,534 @@
+* btn:[Open Project Directory] button with a folder icon, leftmost on the top toolbar.
+--
+====
+
+== Create a new database
+
+To create a new database, follow the steps below.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+
+[,bash]
+----
+typedb console --command="database create try-iam"
+----
+--
+TypeDB Studio::
++
+--
+
+. Open the *Manage Databases* dialog by clicking the database icon on the left side of the toolbar.
++
+image::studio-database.png[Database Manager button,width = 75%,link=self]
+. Enter `try-iam` in the text field, and click the btn:[Create] button next to it.
+. Now select `try-iam` from the database dropdown (next to the database icon).
+--
+====
+
+[#_define_a_schema_from_a_file]
+== Define a schema from a file
+
+Before loading any data into a new database we shall define database's schema.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+
+[,bash]
+----
+typedb console --command="transaction try-iam schema write" --command="source iam-schema.tql" --command="commit"
+----
+--
+TypeDB Studio::
++
+--
+Open the xref:typedb::attachment$iam-schema.tql[iam-schema.tql,window=_blank] file in TypeDB Studio.
+
+Execute the define query from the file:
+
+. Ensure the btn:[Session type] (`schema` / `data`) is set to the `schema` and the
+  btn:[Transaction type] (`write` / `read`) is set to the `write` in the toolbar.
+. Run the query by clicking the btn:[Run Query] button.
+. Commit the changes in transaction by clicking the btn:[Commit Transaction] button.
+--
+====
+
+Schema is defined. The database is ready for inserting data.
+
+== Load data from a file
+
+Let's upload some data into the database.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+
+[,bash]
+----
+typedb console --command="transaction try-iam data write" --command="source iam-data.tql" --command="commit"
+----
+--
+TypeDB Studio::
++
+--
+Open the xref:attachment$iam-data.tql[iam-data.tql] file from the project panel.
+
+Load the data into the database selected earlier, by executing the insert queries from the file:
+
+. Ensure the btn:[Session type] (`schema` / `data`) is set to the `data` and the btn:[Transaction type]
+  (`write` / `read`) is set to the `write` in the toolbar.
+. Execute all queries from the file by clicking the btn:[Run Query] button.
+. Commit the changes in transaction by clicking the btn:[Commit Transaction] button.
+--
+====
+
+Given there were no errors in the process, the data is persisted in the database.
+The database is ready for queries now.
+If you want, try some <<_query_examples,query examples>> below with the new database.
+
+[NOTE]
+====
+The IAM schema and data from this Quickstart guide are widely used in examples throughout the documentation.
+====
+
+[#_query_examples]
+== Query examples
+
+We have prepared a few query examples below:
```

Comment:
Remove.

Reply:
We need to have an introductory sentence for the list